File: php/facadePattern/ClientData.php
<?php    namespace php\facadePattern;

class ClientData {
	private $passenger;
	private $flight;
	private $baggage;
	
   /**
   * Set passenger value
   * 
   * @param string $passenger
   */
    public function setPassenger($passenger) {
		print $this->passenger = $passenger . '<br>';
    }
	
    /**
     * Set flight number
     * 
     * @param string $flight
     */ 
    public function setFlight($flight) {
		print 'Flight: ' . $this->flight = $flight . '<br>';
    }

    /**
     * Set baggage value
     * 
     * @param string $baggage
     */
    public function setBaggage($baggage) {
		print 'Baggage: ' . $this->baggage = $baggage;
    }
}

File: java/facadePattern/ClientData.java
package facadePattern;

public class ClientData {

	public void setPassengerDetails(String[] data) {
		System.out.println("Passenger: " + data[0]);
	}

	public void setFlightDetails(String[] data) {
		System.out.println("Flight: " + data[1]);
	}

	public void setBaggageDetails(String[] data) {
		System.out.println("Baggage: " + data[2]);
	}
}
